src/CookieJar.php

```diff
@@ -310,7 +310,7 @@ public function merge($arg)
 
 		}, func_get_args());
 
-		$data = call_user_func_array('Arr::merge', $arguments);
+		$data = call_user_func_array(__NAMESPACE__.'\Arr::merge', $arguments);
```

Comment:
This is something that we talked about thousand times, but never agreed on something. I think having an Arr instantiated in the constructor will simply work with having a setter. Even better would be to have a getter which instantiates one, if no instance is found.